### PR TITLE
Tech task: Allow setting encryption options

### PR DESCRIPTION
**Pull request by giladshanan**
Allow setting optional encryption method in `ldap.yml`. Valid values: simple_tls or start_tls

### CODE REVIEW

File: vendor/engines/ldap_authentication/lib/ldap_authentication.rb [hunk 1]
```diff
@@ -39,6 +41,18 @@ def self.additional_user_attributes
     config.fetch("additional_user_attributes", [])
   end
 
+  def self.additional_user_attributes
```
Comment (rdunlop) [inline vendor/engines/ldap_authentication/lib/ldap_authentication.rb:44]: this looks like a bad merge..duplicating the lines above it.

**Review by rdunlop — CHANGES_REQUESTED** [review]

File: config/ldap.yml.template [hunk 2]
```diff
@@ -13,3 +13,4 @@ development:
   admin_user: "cn=admin,dc=example,dc=org"
   admin_password: admin
   additional_user_attributes: [] # attributes to update on log in
+  encryption_method: "" # either simple_tls or start_tls
```
Comment (giladshanan) [inline config/ldap.yml.template:16]:
```suggestion
  encryption_method: "" # optional - valid options are either simple_tls, start_tls, or ''
```

**Review by rdunlop — APPROVED** [review]

**Comment by joshea0** [comment]
I realize this is already merged, but I just took a look and this looks great! Thanks @giladshanan for figuring this out. This is a big improvement from my POV

**Comment by giladshanan** [comment]
@joshea0 sure thing! I'm glad we found a way to get this working, and this will be more flexible for future use as well.

**Pull request merged**